smpp/transmitter.go

```diff
@@ -152,6 +177,8 @@ const (
 type ShortMessage struct {
 	Src      string
 	Dst      string
+	DstList  []string // List of destination addreses for submit multi
```

Comment:
Do we want this here? Maybe we want a separate struct `ShortMessageList` or something, to keep things clean.